src/services/assetService.ts

```diff
@@ -192,8 +192,8 @@ export class AssetService {
             if (!types) {
                 console.error(interpolate(strings.editorPage.assetWarning.incorrectFileExtension.failedToFetch, { fileName: corruptFileName.toLocaleUpperCase() }));
             }
-            // If file was renamed/spoofed - fix file extension to true MIME type and show message
-            else if (!types.includes(assetFormat)) {
+            // If file was renamed/spoofed - fix file extension to true MIME if it's type is in supported file types and show message
+            else if (types[0] && !types.includes(assetFormat)) {
```

Comment:
Is it safer to check the length of array?

Reply:
done